static/js/components/widgets/WebsiteCollectionField.tsx

```diff
@@ -36,17 +36,29 @@ export default function WebsiteCollectionField(props: Props): JSX.Element {
     [onChange, websiteMap, name]
   )
 
-  const { options, loadOptions } = useWebsiteSelectOptions("url_path", true)
+  const optionsWithoutShortID = useWebsiteSelectOptions(
+    "title",
+    "",
+    "url_path",
+    true
+  ).options
```

Comment:
Slight issue here: The sortable select is a searchable field:
- Initially, 10 websites are fetched
- if you type, new websites are fetched to match your search filter.

`options` below will be updated when you type. But `optionsWithoutShortID` is never updated.

So I don't think this approach works. 😦 Searching for a course and selecting it, and you'll see its title is not used.

Reply:
`WebsiteCollectionField` maintains a map `websiteMap` of id -> title. Let's call it `intialWebsitesMap`. One idea for addressing this would be to make `useWebsiteSelectOptions` return its `websiteMap` of the loaded objects. 

That's the only idea I can think of without changing the data as discussed in https://github.com/mitodl/ocw-studio/issues/393